packages/examples/src/control-options.ts

```diff
@@ -137,6 +137,14 @@ export const extendedSchema = {
     hideRequiredAsterisk: {
       type: 'string',
       description: 'Hides the "*" symbol, when the field is required',
+    },
+    checkbox: {
+      type: 'boolean',
+      description: 'Checkbox is the default control for boolean values.'
+    },
```

Comment:
We already have a `boolean`/`checkbox` in the first `schema`. No need to re-add it here in `extendedSchema`.

Reply:
That's true but I'm concerned this might break the jsonforms website. There we use the base example and the one with options separately. Thus, I imagine we only use the `extentedData` and `extendedSchema` there. But I can also remove it.

Reply:
That's true :) I meant to remove both, the `checkbox` property and its `Control`. We don't need to show the normal checkbox in the `extendedSchema` and `extendedUischema` as its already a part of the normal `schema` and `uischema`.

Reply:
I see, that makes sense. I'll remove it :)